content/blog/16-17/linux.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Linux: hoe doe?"
+created_at: 02-02-2017
+description: Uitleg en tips voor iedereen die linux wil installeren.
+author: Rien
+---
+
+_Tijdens het begin van ieder schooljaar komt er een verse lading studentjes Informatica toe die in de lessen Computergebruik kennis maken met de wondere wereld van GNU/Linux. Velen onder hen worden geprikkeld door de onontgonnen mysteries van dit besturingssysteem en groeien een verlangen om de kunst van het Unix-tovenaarschap zich eigen te maken._
+
+Deze blogpost dient als een leidraad voor de dappersten onder hen: zij die het pad naar de verlossing willen bewandelen en een eigen Linux-installatie willen tot leven wekken.
+
+# Enkele tips voor je begint
+---
+
+## Probeer eerst in een virtuele machine
+Als je nog geen ervaring hebt met het installeren van Linux start je best door eens te oefenen op een VM (virtuele machine). Zo komt niet alles in één keer op je af. Zo kun je eerst het installatieproces onder de knie krijgen zonder meteen rekening te hoeven houden met eventuele problemen, dual boot opzetten en andere obstakels.
```

Comment:
zonder dat je jezelf in de problemen kunt brengen doordat je naast het OS installeren ook rekening moet houden met andere ostakels zoals een dual-boot systeem opzetten.